pkg/host/internal/udev/udev.go

```diff
@@ -126,6 +127,18 @@ func (u *udev) LoadUdevRules() error {
 	return nil
 }
 
+// WaitUdevEventsProcessed calls `udevadm settle“ with provided timeout
+// The command watches the udev event queue, and exits if all current events are handled.
+func (u *udev) WaitUdevEventsProcessed(timeout int) error {
+	log.Log.V(2).Info("WaitUdevEventsProcessed()")
+	_, stderr, err := u.utilsHelper.RunCommand("udevadm", "settle", "-t", strconv.Itoa(timeout))
```

Comment:
Is this supposed to run in the host chroot? I tried running it in the config-daemon pod:
```
[root@cnfdr22 /]# udevadm settle -t 10
Running in chroot, ignoring request.
```

Reply:
Got it, I wrongly thought the systemd service was running the config-daemon via `podman run ...` .
thanks for explaining